shared/source/os_interface/driver_info.h

```diff
@@ -7,23 +7,54 @@
 
 #pragma once
 
+#include <cstdint>
 #include <string>
 
 namespace NEO {
 
 struct HardwareInfo;
 class OSInterface;
 
+struct PhysicalDevicePciBusInfo {
+    PhysicalDevicePciBusInfo(uint32_t domain, uint32_t bus, uint32_t device, uint32_t function)
+        : pciDomain(domain), pciBus(bus), pciDevice(device), pciFunction(function) {}
+
+    PhysicalDevicePciBusInfo(uint32_t bus, uint32_t device, uint32_t function)
+        : pciDomain(0), pciBus(bus), pciDevice(device), pciFunction(function) {}
+
+    PhysicalDevicePciBusInfo()
+        : PhysicalDevicePciBusInfo(InvalidValue, InvalidValue, InvalidValue, InvalidValue) {}
+
+    bool operator==(const PhysicalDevicePciBusInfo &rhs) const {
+        return pciDomain == rhs.pciDomain && pciBus == rhs.pciBus && pciDevice == rhs.pciDevice && pciFunction == rhs.pciFunction;
+    }
+
+    bool operator!=(const PhysicalDevicePciBusInfo &rhs) const {
```

Comment:
please do not use opeartor overload

Reply:
The operator has been removed from `PhysicalDevicePciBusInfo` class.